telegram_phone_number_checker/main.py

```diff
@@ -80,6 +90,32 @@ async def get_names(client: TelegramClient, phone_number: str) -> dict:
                     "phone": user.phone,
                 }
             )
+        if download_profile_photos is True:
+            try:
+                photo_output_path = Path("{}_{}_photo.jpeg".format(user.id, user.phone))
```

Comment:
It looks like `user.phone` is often `None`, let's use the user-supplied `phone_number` instead

Reply:
(there are a couple of other instances in the block, not just here)

Reply:
f049059c2f0d8b79a844fb6f9253e0169bd66f34